mono/utils/mono-threads-debug.h

```diff
@@ -7,7 +7,7 @@
 
 /* Logging - enable them below if you need specific logging for the category you need */
 #define MOSTLY_ASYNC_SAFE_FPRINTF(handle, ...) do { \
-	char __buff[1024];	\
+	char __buff [128];	\
```

Comment:
How does this impact all the other uses of `MOSTLY_ASYNC_SAFE_FPRINTF`/`MOSTLY_ASYNC_SAFE_PRINTF`. Just a cursory glance should be enough, but there are more uses of it than the crash reporter.

The other solution is to make this a `static inline` function.

Reply:
Yeah, I was careful to check that it worked everywhere (there was one printf that had to be broken up when I increased the usage). Doing it as an inline function would probably be a better idea though. I'll roll out that fix shortly.